frontend/packages/citizen-frontend/src/messages/const.ts

```diff
@@ -0,0 +1 @@
+export const messagesBreakpoint = '850px'
```

Comment:
Where does this breakpoint come from? I personally think it would make sense to have the breakpoint at phone widths. Tablets should be able to show both the message and the list simultaneously.

Reply:
When I was resizing window this was about where the right side part was starting to get awkwardly narrow. To make this breakpoint lower the left side message list would need to somehow shrink (now it is 400px wide).

Reply:
Yeah that could be done by giving the left side a larger `flex-grow` value and setting a `max-width` and then setting a `min-width` for right side so that their sum is the breakpoint.

Reply:
Now I think it works, but the css might not be the minimum and cleanest solution to achieve that.